### ISSUE TITLE
Support Ivy

### ISSUE THREAD

**Issue by dkimmich-onventis** [phrase/ngx-translate-phraseapp#29]
Since Angular 13, when using this package, during build the following message appears:

```
Processing legacy "View Engine" libraries:
- ngx-translate-phraseapp [es2015/esm2015] (https://github.com/phrase/ngx-translate-phraseapp.git)
Encourage the library authors to publish an Ivy distribution.
```

Would be great if you could provide an Ivy distribution of this package.

**Comment by Kjaer** [comment]
Hey @dkimmich-onventis  thanks for your PR (#30) We need to check if there are older version users still relying on our library here. Afterwards we will continue your PR.

**Comment by dkimmich-onventis** [comment]
@Kjaer Thanks for the reply. I understand your concerns regarding backwards compability. For me it is fine to have a "side" / legacy release which still supports the ViewEngine, as long as there is a stable release available with an Ivy distribution. Maybe you could release version 1.0.0 including my changes and keep 0.x with ViewEngine support?

**Comment by dkimmich-onventis** [comment]
@Kjaer any update? 2 more months have passed now 😅

**Comment by Kjaer** [comment]
So sorry for delay, I was hoping, somebody in the Phrase picked up this so far, As I followed their suggestion of "we have better job to do" when I brought this, then found a better job :)

**Comment by forelabs** [comment]
@Varpusparvi Can you or someone of the team take over?

**Comment by Varpusparvi** [comment]
@dkimmich-onventis Is this message causing any problems for your build or workflow?

Right now I would suggest forking this and using your fix for this since we are not sure if we can plan any time for this at the moment.

**Comment by dkimmich-onventis** [comment]
@Varpusparvi I don't have major issues at the moment, just the message being displayed and probably some additional build time. I'm not sure if you're aware that I already created PR #30 which fixes the issue, so nothing to be done at your side besides merging and releasing.

**Comment by Varpusparvi** [comment]
@dkimmich-onventis Released new version 1.0.0, hope this works for you

**Comment by dkimmich-onventis** [comment]
@Varpusparvi thank you very much! With v1.0.0, the message is gone.
